staging/src/k8s.io/kms/service/service.go

```diff
@@ -0,0 +1,116 @@
+package service
+
+import (
+	"context"
+	"encoding/base64"
+
+	"k8s.io/klog/v2"
+	api "k8s.io/kms/apis/v2alpha1"
+	"k8s.io/kms/encryption"
+)
+
+const (
+	version              = "v2alpha1"
+	encryptedLocalKEKKey = "encryptedLocalKEK"
```

Comment:
We validate if annotation keys are fully qualified domain name https://github.com/kubernetes/kubernetes/blob/f14ebac384ffd0a7adc09e5098b0c567042390a0/staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/kmsv2/envelope.go#L293, so this needs to be updated to a valid name